package.json

```diff
@@ -65,9 +65,11 @@
     "react-dnd": "10.0.2",
     "react-dnd-html5-backend": "10.0.2",
     "react-dom": "16.14.0",
+    "react-draggable": "^4.4.6",
     "react-helmet": "6.1.0",
     "react-json-view": "1.21.3",
     "react-popper-tooltip": "4.4.2",
+    "react-resizable": "^3.0.5",
```

Comment:
```suggestion
    "react-draggable": "4.4.6",
    "react-helmet": "6.1.0",
    "react-json-view": "1.21.3",
    "react-popper-tooltip": "4.4.2",
    "react-resizable": "3.0.5",
```

Reply:
Can you just make the changes this way, or would you have to run `npm i` with the exact version number to propagate the changes to package-lock, if any?

Reply:
I believe running `npm i` is still required after applying this change.

Reply:
So this suggested change may be incomplete and should not be committed. because it is missing the package-lock changes - if there are any. New dependencies should be installed with `npm i -E react-draggable@4.4.6` instead of manually changing the version in the package.json.

Reply:
@vardhan0604 To fix this, could you please run and commit?

`npm i -E react-draggable@4.4.6 react-resizable@3.0.5`